File: games/planetization/bots/java/src/core/api/InitialData.java
package core.api;

import core.MatchDetails;

import java.util.ArrayList;

public class InitialData {
    public int __uid;
    public MatchDetails __matchDetails;

    public int mapWidth;
    public int mapHeight;
    public float unitSize;
    public float planetDiameter;
    public float unitSpeed;
    public float unitRotationSpeed;
    public int numberOfWorkersOnStart;
    public int unitCost;
    public float resourceGenerationSpeed;
    public int maxActiveWorkersPerPlanet;
    public int maxNumberOfUnitsPerTeam;
    public float maxMatchDuration;
    public float workerHealth;
    public float workerAttack;
    public float warriorHealth;
    public float warriorAttack;
    public float damageReductionRatioOnDefence;

    public ArrayList<PlanetData> yourPlanets;
    public ArrayList<PlanetData> freePlanets;
    public ArrayList<PlanetData> opponentPlanets;
    public ArrayList<UnitData> yourWorkers;
    public ArrayList<UnitData> opponentWorkers;
    public ArrayList<UnitData> yourWarriors;
    public ArrayList<UnitData> opponentWarriors;
}

File: games/planetization/bots/kotlin/src/core/api/InitialData.kt
package core.api

import core.MatchDetails
import java.util.ArrayList

class InitialData(
        var __uid: Int,
        var __matchDetails: MatchDetails,
        var mapWidth: Int,
        var mapHeight: Int,
        var unitSize: Float,
        var planetDiameter: Float,
        var unitSpeed: Float,
        var unitRotationSpeed: Float,
        var numberOfWorkersOnStart: Int,
        var unitCost: Int,
        var resourceGenerationSpeed: Float,
        var maxActiveWorkersPerPlanet: Int,
        var maxNumberOfUnitsPerTeam: Int,
        var maxMatchDuration: Float,
        var workerHealth: Float,
        var workerAttack: Float,
        var warriorHealth: Float,
        var warriorAttack: Float,
        var damageReductionRatioOnDefence: Float,

        var yourPlanets: ArrayList<PlanetData>,
        var freePlanets: ArrayList<PlanetData>,
        var opponentPlanets: ArrayList<PlanetData>,
        var yourWorkers: ArrayList<UnitData>,
        var opponentWorkers: ArrayList<UnitData>,
        var yourWarriors: ArrayList<UnitData>,
        var opponentWarriors: ArrayList<UnitData>
)
